insights/combiners/du.py

```diff
@@ -0,0 +1,43 @@
+"""
+Disk Usages Combiners
+=====================
+
+Combiners for gathering information from du parsers.
+"""
+
+from insights import combiner
+from insights.parsers.du import DiskUsageDir
+
+
+@combiner(DiskUsageDir)
+class DiskUsageDirs(dict):
+    """
+    Combiner for the :class:`insights.parsers.du.DiskUsageDir` parser.
+
+    The parser is multioutput, one parser instance for each directory disk
+    usage. This combiner puts all of them back together and presents them as a
+    dict where the keys are the directory names and the space usage are the
+    values.
+
+    Sample input data for du commands as parsed by the parsers::
+
+    # Output of the command:
+    # /bin/du -s -k /var/log
+    553500	/var/log
+
+    # Output of the command:
+    # /bin/du -s -k /var/log
```

Comment:
Should this be `/var/lib`?  Looks like a cut-and-paste error.  Also the example input data needs to be indented.  Please following the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-docstring) to generate the documentation output and check it for errors.

Reply:
Apology, Fixed flake8, I will be careful from next time.

Fixed the typo, /var/log to /var/lib/pgsql

Also did some minor doc changes.